src/components/BulkEnrollmentPage/CourseSearchResults.test.jsx

```diff
@@ -107,17 +118,17 @@ describe('<CourseSearchResults />', () => {
   it('renders search results', () => {
     const wrapper = mount(<CourseSearchWrapper />);
 
-    // Four header columns, one for sorting, one for Course Name, one for Course Run, one for the enroll column
+    // 5 header columns: selection, Course name, Partner, Course Date, and enrollment
```

Comment:
This comment was confusing the hell out of me since it didn't match the header names nor count. If there is a reason it was labeled like this I can put it back, but it was just really, REALLY confusing.

Reply:
Do we need it at all?

Reply:
Not particularly. 🤷  I just figured if it was going to exist it should at least make sense lmao